lib/queuePopulator/LogReader.js

```diff
@@ -226,6 +241,7 @@ class LogReader {
                 this.log.info('replication batch finished', {
```

Comment:
I think you need to rebase, I recently modified the message `replication batch finished` to `batch finished` or you could resolve conflict :)

Reply:
Ah, I think it will be alright the merge should be automatic. If need I'll do it for sure.